presto-main/src/main/java/com/facebook/presto/sql/planner/PlanPrinter.java

```diff
@@ -1213,7 +1214,7 @@ private void printAssignments(Assignments assignments, int indent)
 
         private String formatOutputs(Iterable<Symbol> symbols)
         {
-            return Joiner.on(", ").join(Iterables.transform(symbols, input -> input + ":" + types.get(input).getDisplayName()));
+            return Joiner.on(", ").join(Iterables.transform(symbols, input -> input + ":" + types.getOrDefault(input, UNKNOWN).getDisplayName()));
```

Comment:
So if the test is now defined, you can switch back to:
```
types.get(input)
```

Assumption that when there is no type in map, then the type is `UNKNOWN` is incorrect. There is difference between no type computed and computed `UNKNOWN` type